### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Arnab-Developer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # demo2
 date
+10 05
```
Comment (arnab-jit1) [inline README.md]: provide proper date format

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # demo2
 date
+10 May 2024
```
Comment (arnab-jit1) [inline README.md:3]: date format is fine but add time also

File: README.md (hunk 1 above)
Comment (Arnab-Developer) [reply README.md]: date format added

File: README.md (hunk 2 above)
Comment (Arnab-Developer) [reply README.md]: time added

File: README.md [hunk 3]
```diff
@@ -1,2 +1,3 @@
 # demo2
 date
+10 May 2024 10:30 am
```
Comment (arnab-jit1) [inline README.md:3]: add newline

Comment (Arnab-Developer) [reply README.md:4]: done

**Review by arnab-jit1 — APPROVED** [review]

**Pull request merged**